<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BLOCKEDITOR</class>
 <widget class="QMainWindow" name="BLOCKEDITOR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <action name="openAct">
   <property name="text">
    <string>&amp;Otvoriť</string>
   </property>
   <property name="toolTip">
    <string>Otvoriť</string>
   </property>
  </action>
  <action name="saveAct">
   <property name="text">
    <string>&amp;Uložiť</string>
   </property>
   <property name="iconText">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Uloží vykonané zmeny</string>
   </property>
  </action>
  <action name="saveAsAct">
   <property name="text">
    <string>Uložiť ako</string>
   </property>
   <property name="toolTip">
    <string>Uložiť ako</string>
   </property>
  </action>
  <action name="helpAct">
   <property name="text">
    <string>Nápoveda</string>
   </property>
   <property name="toolTip">
    <string>Nápoveda</string>
   </property>
  </action>
  <action name="aboutAct">
   <property name="text">
    <string>O programe</string>
   </property>
   <property name="toolTip">
    <string>O programe</string>
   </property>
  </action>
  <action name="exitAct">
   <property name="text">
    <string>Zatvoriť</string>
   </property>
   <property name="toolTip">
    <string>Zatvoriť</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Zatvoriť</string>
   </property>
   <property name="toolTip">
    <string>Zatvoriť</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
